index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Fred's Personal Portfolio</title>
+</head>
+<body>
+    <div class="header-container">
+        <header>
+            <nav class="nav-menu">
+                <a href="" class="logo"><img src="images/My-Logo.svg" alt=""></a>
+                <a href="" class="menu-icon"><img src="images/Icon-Menu.svg" alt=""></a>
+            </nav>
+            <div class="main-heading">
+                <h1>I’m Mariane Glad to see you!</h1>
+            </div>
```

Comment:
- [ ] Kindly fix the spacing around your h1 tag since it's overlapping with the text that follows.
![text](https://user-images.githubusercontent.com/69858269/164504850-b9a18a12-9576-4a40-977c-dfc9ad741143.PNG)